src/app/redux/TransactionSaga.js

```diff
@@ -467,6 +467,8 @@ export function* createPermlink(title, author, parent_author, parent_permlink) {
         parent_permlink = parent_permlink.replace(/(-\d{8}t\d{9}z)/g, '');
         permlink = `re-${parent_author}-${parent_permlink}-${timeStr}`;
     }
+    // only letters numbers and dashes shall survive
+    permlink = permlink.toLowerCase().replace(/[^a-z0-9-]+/g, '');
```

Comment:
If the only issue here is that parent can have a period in the name, how about we remove this line and just replace out any `.` of the `parent_author` in the `else` branch above? This keeps it minimal and explicit.

Reply:
Yes I can do that. I think I would still remove all of them though. Are there any other characters allowed in the account name? I was thinking dashes, but now I see that dashes are included in the permlink list.

Reply:
Period is the only character allowed in account name that's not allowed in permlink :)

Reply:
Ok. will change. One moment